pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -65,15 +65,14 @@ theme: responsive
  </tr>
 </table>
 
-
 ## Sites to Unblock
 
 For the very best experience with all Code.org content, we recommend consulting with your school or district's IT department to ensure the following sites are allowed and aren't blocked.
 
 | Priority                                                                                 | URL                                                                                                                                                    |
-| ---------------------------------------------------------------------------------------- |--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ---------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | **Required**                                                                             | `https://code.org`<br/>`https://*.code.org`                                                                                                            |
-| **Optional**                                                                             | `https://cdn.optimizely.com`<br/>`https://www.google-analytics.com`<br/>`https://www.googleoptimize.com`                                               |
+| **Optional**                                                                             | `https://www.google-analytics.com`<br/>`https://www.googleoptimize.com`                                                                                |
```

Comment:
tiny nit: is the extra whitespace needed?

Reply:
This support page will also need to be updated: https://support.code.org/hc/en-us/articles/202591743-Technical-requirements-for-Code-org

Reply:
Oh, I was thinking that was being generated from this file, but now I realize I think it's a different one. Will update.